dxr/vcs.py

```diff
@@ -327,8 +329,125 @@ def get_contents(cls, path, revision, stderr=None):
                                         filename + '@' + revision],
                                        cwd=directory, env=env, stderr=stderr)
 
+class Subversion(Vcs):
+    command = 'svn'
+
+    def __init__(self, root):
+        super(Subversion, self).__init__(root)
+        # All tracked files as a dictionary (file => revision).
+        self.files = self.__get_files_with_revision()
```

Comment:
get_files_with_revision requires a path parameter

Reply:
Actually, my implementation of `_get_files_with_revision` does not require a path parameter, see here:
https://github.com/mozilla/dxr/pull/580/files#diff-fdc95e419de8a362ed3a4518be94408bR437